src/commands/init.js

```diff
@@ -32,56 +32,41 @@ class InitCommand extends Command {
       },
     })
 
-    // const hasFlags = !isEmpty(flags)
-    let siteData
-    try {
-      siteData = await api.getSite({ siteId })
-    } catch (error) {
-      // silent api error
-      // TODO handle expected errors
-      // Throw unexpected ones
-    }
-
     // Add .netlify to .gitignore file
     await ensureNetlifyIgnore(site.root)
 
-    if (siteId && siteData && get(siteData, 'build_settings.repo_url') && !flags.force) {
-      const repoUrl = get(siteData, 'build_settings.repo_url')
+    const repoUrl = get(siteInfo, 'build_settings.repo_url')
+    if (repoUrl && !flags.force) {
       this.log()
       this.log(`This site has been initialized`)
       this.log()
-      this.log(`Site Name:  ${chalk.cyan(siteData.name)}`)
-      this.log(`Site Url:   ${chalk.cyan(siteData.ssl_url || siteData.url)}`)
-      if (repoUrl) {
```

Comment:
This condition was redundant since it was check already here:
https://github.com/netlify/cli/pull/1607/files#diff-d9c0f8c5c3ab192e3bbc073115e4f12d530bcad9db684563da982b0767c5a94fL48

and since `repoUrl === get(siteData, 'build_settings.repo_url')`